staging/src/github.com/kubeedge/api/apis/componentconfig/edgecore/v1alpha2/types.go

```diff
@@ -99,6 +99,10 @@ type Edged struct {
 	// if set to false (for debugging etc.), skip checking other EdgeHub configs.
 	// default true
 	Enable bool `json:"enable"`
+	// SendEvents indicates whether to send events to cloud,
+	// if set to false, events will not be sent to cloud.
+	// default true
+	SendEvent bool `json:"sendEvent"`
```

Comment:
In order to allow users to smoothly upgrade and adapt to new versions,  I think it is more appropriate to disable this feature by default in the next two release.